mlflow/utils/credentials.py

```diff
@@ -154,10 +160,16 @@ def _overwrite_or_create_databricks_profile(
 
 
 def _databricks_login():
-    """Set up databricks authentication and connect MLflow to Databricks tracking server."""
-    if _check_databricks_auth():
-        # Check if the auth has already been set.
+    """Set up databricks authentication."""
+    try:
+        # Failed validation will throw an error.
+        _validate_databricks_auth()
         return
+    except Exception:
+        # If no valid auth is found, we will prompt the user to enter thepy auth.
+        pass
+
+    _logger.info("No valid Databricks credentials found, creating a new one...")
```

Comment:
/ai I'd like to replace `creating a new one...` with a different phrase. This message is printed out before asking a user for credetials.

Reply:
/ai doesn't sound ba. Suggest more messages.